src/encoded/audit/experiment.py

```diff
@@ -1329,6 +1331,13 @@ def check_experiment_micro_rna_standards(
     upper_limit_expressed_mirnas,
     lower_limit_expressed_mirnas,
 ):
+    # Audit read length
+    minimum_threshold = 30 
+    for f in fastq_files:
+        yield from check_file_read_length_rna(f, minimum_threshold,
+                                                  'microRNA-seq pipeline',
+                                                  experiment['assay_term_name'],
+                                                  '/microrna/microrna-seq/')
```

Comment:
The last argument here should following other function as being passed over here from `standards_links[pipeline_title]`.